hack/dind-cluster.sh

```diff
@@ -125,8 +133,15 @@ function start() {
 
   if [[ -n "${ADDITIONAL_NETWORK_INTERFACE}" ]]; then
     add-network-interface-to-nodes "${config_root}"
-    update-master-config
-    update-node-config
+  fi
+
+  if [[ -n "${local_registry}" ]]; then
+    add-registry-to-runtime "${registry_ip}:${REGISTRY_PORT}" "${container_runtime}"
```

Comment:
I think we need to detect if this has already been done

Reply:
We need to detect if registry entries already exists if adding local registry was a separate dind call. In here, we added local registry as an optional param to dind start. So we are guaranteed that registry entries will not present beforehand. 
dind nodes and master are freshly created just few lines above this call. So local registry entries will not be present.